pymongo/encryption_options.py

```diff
@@ -55,9 +55,8 @@ def __init__(self, kms_providers, key_vault_namespace,
         See :ref:`automatic-client-side-encryption` for an example.
 
         :Parameters:
-          - `kms_providers`: Map of KMS provider options. Two KMS providers
-            are supported: "aws" and "local". The kmsProviders map values
-            differ by provider:
+          - `kms_providers`: Map of KMS provider options.The kmsProviders map
```

Comment:
Missing space before the second sentence?

Reply:
Done.